app/src/main/java/com/ucmobiledevelopment/freeways/service/IncidentService.kt

```diff
@@ -22,17 +22,17 @@ class IncidentService : IIncidentService {
     override suspend fun fetchIncidents(fromCaseYear: Int, toCaseYear: Int, state: Int, county: Int) : List<Incident>? {
 
         return withContext(Dispatchers.IO) {
-            var incidentList: MutableList<Incident> = mutableListOf()
+            val incidentList: MutableList<Incident> = mutableListOf()
 
             val service = RetrofitClientInstance.retrofitInstance?.create(IIncidentDAO::class.java)
             val incidentListDTO =
                 async { service?.getAllIncidents(fromCaseYear, toCaseYear, state, county) }
-            var result = incidentListDTO.await()?.awaitResponse()?.body()
+            val result = incidentListDTO.await()?.awaitResponse()?.body()
 
             if (result != null) {
                 if(result.Results[0] != null){
                     result.Results[0].forEach{
-                        var newIncident: Incident = Incident()
+                        val newIncident: Incident = Incident()
```

Comment:
The vars did not contain mutable data, so I changed them to val. This may not work for what you guys are trying to do with the code but for now I have changed it.

Reply:
Good!